HighwayPatrol.cpp

```diff
@@ -28,10 +28,23 @@ void HighwayPatrol::pullOver( Vehicle* v, bool willArrest, Highway* h )
     std::cout << name << ": vehicle is traveling " << v->speed - h->speedLimit << " miles per hour over the speed limit" << std::endl;
     if( willArrest )
     {
-        assert(false);
+        //assert(false);
         //print the vehicle type in this std::cout between "THE [" and "] PULL". 
-        std::cout << name << ": YOU IN THE [ " << " ] PULL OVER AND SHOW YOUR HANDS" << std::endl;
+        std::cout << name << ": YOU IN THE [ " << v->name << " ] PULL OVER AND SHOW YOUR HANDS" << std::endl;
         std::cout << "EVERYONE ELSE, SLOW DOWN!! \n\n\n";
         h->removeVehicle(v);
     }
 }
+
+int HighwayPatrol::checkSpeed(Vehicle* v)
+{
+    int vSpeed = 0;
+    v->speed = vSpeed;
+    return vSpeed;
```

Comment:
This is not how to check the speed.
This is setting the speed of each vehicle.
Also, the instructions do not tell you to check the speed.
- [ ] remove this function